environment/container/podmanengine.go

```diff
@@ -38,15 +38,15 @@ func newPodmanEngine() *podmanEngine {
 }
 
 // InspectImage returns inspect output for an image using Podman
-func (e *podmanEngine) InspectImage(image string) (t types.ImageInspect, err error) {
+func (e *podmanEngine) InspectImage(image string) ([]types.ImageInspect, error) {
 	inspectcmd := exec.Command("podman", "inspect", image)
 	logrus.Debugf("Inspecting image %s", image)
 	output, err := inspectcmd.CombinedOutput()
 	if err != nil {
 		logrus.Debugf("Unable to inspect image %s : %s, %s", image, err, output)
-		return t, err
+		return nil, err
 	}
-	t = types.ImageInspect{}
+	var t []types.ImageInspect
```

Comment:
Inspect functionality in podman returns an array of json, which can only be unmarshalled into a slice of structs.

Fixes this: 
<img width="1306" alt="Screenshot 2023-03-26 at 11 37 29 PM" src="https://user-images.githubusercontent.com/40015660/227795356-c026ee1a-9ec2-4647-9d02-f9acb633d81f.png">